bridge/bridge.go

```diff
@@ -198,6 +198,14 @@ func (b *Bridge) add(containerId string, quiet bool) {
 		return
 	}
 
+	if b.config.FilterRegexp) != "" {
```

Comment:
```suggestion
	if b.config.FilterRegexp != "" {
```
Wouldn't docker build for me otherwise